docs/docs/integrations/chat/google_vertex_ai_palm.ipynb

```diff
@@ -57,8 +56,10 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 1,
```

Comment:
can we add a specific note that `ChatVertexAI` support Gemini and Palm on top of the documentation page, please?